README.adoc

```diff
@@ -847,6 +847,28 @@ else
 end
 ----
 
+=== Nested Transactions [[nested-transactions]]
```

Comment:
Similar to what I said in the corresponding RuboCop PR, I think we could group this into a section about avoiding private APIs.

Reply:
Thank you for your suggestion.

However, there may be different reasons why private APIs should not be used, so I'm wondering if it's appropriate to group them all in one section.

Reply:
I think the section should remain as it is because the focus should be more on the issues that arise with nested transactions rather than the use of private APIs.

@wata727 I’ve been reading your blog, and I was wondering if you could elaborate a bit more on `after_commit` in this style guide?

Reply:
I'm thinking about this PR again now, and I'm beginning to think that the section might not be suitable as a "style guide." I believe it’s worth having RuboCop find the problems caused by `joinable: false`, but it might be endless to list all such problems in the style guide.

@koic Is there a chance we could add a cop without adding it to the style guide, or do you think problems like this are worth mentioning in the style guide?